CIPs/0020.md

```diff
@@ -0,0 +1,167 @@
+# CIP [0020]: Generic hash function support via extensible precompile
+
+---
+cip: 20
+title: Generic hash function support via extensible precompile
+author: James Prestwich <james@prestwi.ch>
+discussions-to: <URL>
```

Comment:
Hi @prestwich can you add a url link? It can be either a Github issue called "Discussions for Extensible precompile for hash functions" or something on the forum.celo.org page.